lib/modules/apostrophe-ui/public/css/components/fields.less

```diff
@@ -33,6 +33,12 @@
   padding: @apos-padding-1 @apos-padding-2;
 }
 
+.apos-empty-widget(@minHeight: 100px) {
```

Comment:
This should move to `lib/modules/apostrophe-ui/public/css/mixins/utils.less`. The areas module user.less file isn't finding it.

Reply:
The style is missing text centering. It should be fine to make it `display: flex` and use flex properties to center.

Reply:
Ah, thank you! I was testing last night trying to find out why the areas file wasn't able to find it. Good to know there's a dedicated mixins file.

The default styling for the existing class doesn't center the text - is it okay to change the rich-text empty area to be centered as well? I guess it should be fine as there isn't usually text shown there.

Reply:
Yeah, exactly. If we ever did add placeholder text in there we'd want it centered as well.

Reply:
Great, thank you. I just discovered NPM link, which should help in the future - I was pushing my code just so I could test it in a separate project, not realizing there was another way to pull in another NPM package locally. Next time I'll be able to test prior to pushing, which would probably help for things like this 😛. Thanks again, I'll have all of these items fixed shortly.

Reply:
Oh yes! `npm link` is crucial for this kind of thing. It'll be a game changer.